src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -7919,6 +7919,8 @@ private static void tryFinallyChecks(MethodHandle target, MethodHandle cleanup)
      *                                  handles is not {@code int}, or if the types of
      *                                  the fallback handle and all of target handles are
      *                                  not the same.
+     *
+     * @since 17
      */
     public static MethodHandle tableSwitch(MethodHandle fallback, MethodHandle... targets) {
```

Comment:
This method was introduced in Java 17 through https://bugs.openjdk.org/browse/JDK-8263087. So this addition of `@since 17` looks fine to me.